src/core/mod.rs

```diff
@@ -257,14 +257,16 @@ impl TracingPublicKey {
 /// - the scalar `s` used to bind tracing and coordinate keys;
 /// - the tracing secret key used to produce challenges to trace user keys;
 /// - the secret keys associated to the universal coordinates;
-/// - an optional key for symmetric USK-signing.
+/// - an optional key for symmetric USK-signing;
+/// - the policy defined for the user.
 #[derive(Debug, PartialEq, Eq)]
 pub struct MasterSecretKey {
     s: Scalar,
     tsk: TracingSecretKey,
     coordinate_secrets: RevisionMap<Coordinate, (bool, CoordinateSecretKey)>,
     signing_key: Option<SymmetricKey<SIGNING_KEY_LENGTH>>,
-    policy: Policy,
+    // ask Théophile 
+    pub policy: Policy,
```

Comment:
You can expose it at the crate level with `pub(crate)`, but we don't want to expose the policy publicly as the whole point of this MR was to internalize it.